### ISSUE TITLE
Pipe / Redirect output to other panes

### ISSUE THREAD

**Issue by imsnif** [zellij-org/zellij#1402]
We should allow these sorts of interfaces

`tail -f "/path/to/my/log" > zc new-pane` (zc will be an alias, short for `zellij command`)
`vim my-file.rs | zc new-floating-pane`

Basically, allow to both pipe and redirect command output to a new or existing pane.

Depends on: #1265

**Comment by jwbowen** [comment]
This would be awesome!

**Comment by zorzysty** [comment]
Would it allow achieving similar experience as `fzf-tmux -p`?

I have this particular scenario in mind:
Inside pane (A) run a command that opens `fzf` in a floating pane (B), then automatically closes pane (B) when fzf is done, but transfers its output back to pane (A).

I know it's only cosmetic, but that's the feature I'm missing the most after moving from tmux to zellij.

**Comment by Dom324** [comment]
Hey @zorzysty , I believe I managed to do what you are describing. Some time ago I looked into `fzf-tmux` and tried to create `fzf-zellij`. It is probably not fully working, but it works for CTRL+R and kinda also for tab, which was enough for me at the time.

![Peek 2023-08-13 18-19](https://github.com/zellij-org/zellij/assets/57506340/54912db2-b56e-47e9-af10-3553c74a8adc)

I could share the sources if someone wants, although it will probably require some tweaking to get it working.
In my opinion it might be worthwhile to get something like this merged into Zellij/fzf itself, since I think a lot of people use the combination of fzf + zellij.

**Comment by llogiq** [comment]
Question: Would this also allow mirroring a pane to multiple screens? Or is this already available by having multiple terminals attach to a single session?

**Comment by imsnif** [comment]
> Question: Would this also allow mirroring a pane to multiple screens? Or is this already available by having multiple terminals attach to a single session?

Right now we can mirror a whole session to another terminal window (and thus another screen) if we start the session with `zellij options --mirror-session true` - then attach to it from the different window.

Mirroring a specific pane only (or a specific tab) is a cool idea.

**Comment by har96** [comment]
The ability to mirror an existing pane would be great. I'd love to be able to quickly pop open a floating pane in one tab that is a mirror for some pane in another tab (e.g. view the compilation output of a pane in my "build" tab in a floating pane one my "editor" tab). I can switch tabs if I need to do some actual work, but I can quickly show/hide that floating pane when I just need to reference the output.
